Client/Frontend/Widgets/ActionOverlayTableViewController.swift

```diff
@@ -75,6 +77,22 @@ class ActionOverlayTableViewController: UIViewController, UITableViewDelegate, U
         tableView.delegate = nil
     }
 
+    override func updateViewConstraints() {
+        tableView.snp_updateConstraints { make in
+            make.height.lessThanOrEqualTo(UIScreen.mainScreen().bounds.size.height).priorityHigh()
+        }
+        super.updateViewConstraints()
+    }
+
+    override func traitCollectionDidChange(previousTraitCollection: UITraitCollection?) {
```

Comment:
Actually, this conditional is a suggested code snippet in the Apple documentation. https://developer.apple.com/reference/uikit/uitraitenvironment/1623516-traitcollectiondidchange

Also, since the same function is called regardless of whether it is landscape or portrait, I think `traitCollectionDidChange` is a good fit. Let me know if you think otherwise.